index.html

```diff
@@ -401,6 +401,11 @@
           href: 'https://www.nytimes.com/2018/01/30/opinion/strava-privacy.html',
           publisher: 'The New York Times',
         },
+        'Standard-Bodies-Regulators': {
+          title: 'Technical Standards Bodies are Regulators',
+          authors: ['Mark Nottingham'],
+          href: 'https://www.mnot.net/blog/2023/11/01/regulators',
```

Comment:
I'm nervous about citing an individual's blog posts. We already have [[New-Chicago-School]] for this citation, and while Mark says the relevant bits more directly, I'd rather use citations with a little more pre-publication review. That said, I'm not firmly opposed to citing this, and I think we have other citations that are just as informal.

Reply:
In my mind the value of citing this is indeed in part that it's more direct but also, rather importantly, that I know that it's written in a way that makes sense to policymakers. It makes W3C make sense with respect to their work. I think that helps with the delicate balance that we're trying to set up here.